apps/emqx_connector/src/emqx_connector_redis.erl

```diff
@@ -185,3 +192,18 @@ to_server(Server) ->
         [Host, Port] -> {ok, {Host, list_to_integer(Port)}};
         _ -> {error, Server}
     end.
+
+to_servers(Servers) ->
+    try {ok, lists:map(fun parse_server/1, string:tokens(Servers, ", "))}
```

Comment:
It seems that the problem with `string:tokens([<<"host1">>, <<"host2">>], ", ")` still exists?